.circleci/config.yml

```diff
@@ -169,9 +164,10 @@ jobs:
     working_directory: /esmvaltool
     docker:
       - image: continuumio/miniconda3
+    resource_class: medium+
```

Comment:
This breaks the circle-ci build for me :-(

Reply:
https://app.circleci.com/pipelines/github/ESMValGroup/ESMValTool/3128/workflows/237313fa-aac4-41da-bbe5-c40f961c40de/jobs/36447

```
This job has been blocked because resource-class:medium+ is not available on your plan. Please upgrade to continue building.
```

Reply:
I removed that a while ago man, why are you still picking it up?

Reply:
meh proves out I didnt - sorry about that @nielsdrost - that test failes in Circle coz 137 (memory), that's why we now have a GA test that does that